frameworks/Scouter-java/labels.sh

```diff
@@ -0,0 +1,4 @@
+TITLE[0]="No instrumentation"
+TITLE[1]="Default Scouter"
```

Comment:
What is "Default Scouter", is it "No instrumentation", "No profiling", ...? It would make sense to choose a title that is as exact as possible (and also comparable to what the other frameworks do).